aws/src/test/java/org/apache/iceberg/aws/TestAwsClientFactories.java

```diff
@@ -56,6 +58,17 @@ public void testLoadCustom() {
         "should load custom class", AwsClientFactories.from(properties) instanceof CustomFactory);
   }
 
+  @Test
+  public void testLoadLakeFormationAwsClientFactory() {
+    AwsClientFactory awsClientFactory = new LakeFormationAwsClientFactory();
+
+    byte[] data = SerializationUtil.serializeToBytes(awsClientFactory);
+    AwsClientFactory expectedAwsClientFactory = SerializationUtil.deserializeFromBytes(data);
+    Assert.assertTrue(
```

Comment:
this is guaranteed if serialization roundtrip succeeds. We just need to run L65-66 and make sure it does not fail.

Reply:
So you suggest to remove this assertion? I feel it's better to still verify something.

Reply:
I see, yeah that works for me